docs/contributing.md

```diff
@@ -17,6 +17,14 @@ It is recommended that automated github actions are turned on in your fork to ha
 You can turn them on here:  
 ![Github actions example](./img/github-actions-example.png)
 
+* **When reviewing PRs, please leave a message what was done.**
+We don't have full test coverage, so manual testing can help.
```

Comment:
It highly depends on the code that was touched - assuming the author tested a complicated change, that is not fully covered by automated testing, then yes, having a second person test as well would be preferred, since the original author might have missed something as they might have a blind spot for things they "didn't touch", but it may not be mandatory. 

In the PR I only specifically mention the case where multiple worlds could be affected. I can try to reword this in a more generic way, if that helps?

Either way the more important thing I am asking there is that a reviewer let us know what they tested or what/how they reviewed. If this is not clear from the text, I'm open for suggestions.

If you disapprove in that it should not be necessary for a peer reviewer to tell us how they reviewed and/or how they came to their conclusion (approval), then please don't expect me to take the waiting-on: core review serious.